base_layer/service_framework/src/tower/service_ext.rs

```diff
@@ -117,10 +117,9 @@ mod test {
 
         impl Service<u32> for ReadyLater {
             type Error = ();
+            type Future = Pin<Box<dyn Future<Output = Result<u32, ()>>>>;
             type Response = u32;
 
-            type Future = impl Future<Output = Result<Self::Response, Self::Error>>;
```

Comment:
For this one we would rather want to static type

```rust
type Future = future::Ready<Result<u32, ()>>;
```

and return `future::ok(...)`

Reply:
Nice thanks I'll try that!

Reply:
Just noticed that this is in a test - so really doesnt matter at all - but thanks for entertaining my nit 😄